src/netmanager-app/components/Organization/organization-roles.tsx

```
@@ -41,169 +30,225 @@
-  }, [grproles, searchQuery, sortField, sortOrder]);
+      })
+  }, [grproles, searchQuery, sortField, sortOrder])
+
+  const filteredPermissions = useMemo(() => {
+    // Ensure allPermissions is an array before filtering
+    if (!Array.isArray(allPermissions)) return []
+
+    return allPermissions.filter(
+      (permission) =>
+        permission.permission.toLowerCase().includes(permissionsSearch.toLowerCase()) ||
+        (permission.description && permission.description.toLowerCase().includes(permissionsSearch.toLowerCase())),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)